src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -156,6 +156,7 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = props => {
         requiresArtsyShipping: requiresArtsyShippingToDestination,
       })
     } catch (error) {
+      console.log("error: ", error)
```

Comment:
can we use our logger as we have elsewhere?

Reply:
oh no. anywhere i leave `console` rather than `logger` in is a debugging oversight. I guess I assumed the linter would flag this for me, sorry.